candy-machine/js/idl/candy_machine.json

```diff
@@ -626,65 +618,73 @@
       }
     },
     {
-      "name": "ConfigLine",
+      "name": "WhitelistMintSettings",
       "type": {
         "kind": "struct",
         "fields": [
           {
-            "name": "name",
-            "type": "string"
+            "name": "mode",
+            "type": {
+              "defined": "WhitelistMintMode"
+            }
           },
           {
-            "name": "uri",
-            "type": "string"
+            "name": "mint",
+            "type": "publicKey"
+          },
+          {
+            "name": "presale",
+            "type": "bool"
+          },
+          {
+            "name": "discountPrice",
+            "type": {
+              "option": "u64"
+            }
           }
         ]
       }
     },
     {
-      "name": "Creator",
+      "name": "GatekeeperConfig",
       "type": {
         "kind": "struct",
         "fields": [
           {
-            "name": "address",
+            "name": "gatekeeperNetwork",
             "type": "publicKey"
           },
           {
-            "name": "verified",
+            "name": "expireOnUse",
             "type": "bool"
-          },
-          {
-            "name": "share",
-            "type": "u8"
           }
         ]
       }
     },
     {
-      "name": "WhitelistMintMode",
+      "name": "EndSettingType",
       "type": {
         "kind": "enum",
         "variants": [
           {
-            "name": "BurnEveryTime"
+            "name": "Date"
           },
           {
-            "name": "NeverBurn"
+            "name": "Amount"
           }
         ]
       }
     },
     {
-      "name": "EndSettingType",
+      "name": "WhitelistMintMode",
       "type": {
         "kind": "enum",
         "variants": [
           {
-            "name": "Date"
+            "name": "BurnEveryTime"
           },
           {
-            "name": "Amount"
+            "name": "NeverBurn"
           }
         ]
       }
```

Comment:
What I'm noticing here is that errors still didn't get fixed.
Couldn't we upgrade to anchor 0.24 to get those proper again?

Reply:
Actually confirmed that errors were indeed already fixed https://github.com/metaplex-foundation/metaplex-program-library/blob/cf472e24686f997eceefc7306b89dd9ea326222a/candy-machine/js/idl/candy_machine.json#L693